forager/observations.py

```diff
@@ -3,10 +3,20 @@
 
 from typing import Dict
 from forager.interface import Coords, Size
-
+from forager.state import ForagerState
+
+def get_color_vision(s: ForagerState):
```

Comment:
Numba can't handle the `ForagerState` object directly, so we use this wrapper pattern to translate from the high-level API to the low-level numba function